src/Tizen.NUI/src/public/KeyValue.cs

```diff
@@ -30,16 +31,25 @@ public class KeyValue : IDisposable
         /// <summary>
         /// Int key.
         /// </summary>
+        /// When deleting the field, change it to prperty.
+        [Obsolete("Deprecated in API9, Will be changed to visible property")]
+        [SuppressMessage("Microsoft.Design", "CA1051:Do not declare visible instance fields")]
```

Comment:
In this case, we can not provide replaceable property. It means application can not fix compile warning. If we can change the filed into Property in this patch, it is best. If we can't we can't deprecate the field.

Reply:
I modified it by adding a setter/getter function. 
@dongsug-song @hyunjushin  How about these modifications?